installation_docs/manual/linux.md

```diff
@@ -0,0 +1,50 @@
+## Installation on Linux
+
+#### Cloning From GitHub
+To clone the repository, either use the Git GUI if you have one installed or enter the following commands:
+```sh
+git clone https://github.com/CircuitVerse/CircuitVerse.git
+cd CircuitVerse
+```
+
+**Note:** If you want to contribute, first fork the original repository and clone your **forked** repository into your local machine. If you don't do this, you will not be able to make commits or change any files.
+```sh
+git clone https://github.com/<username>/CircuitVerse.git
+cd CircuitVerse
+```
+
+#### Dependencies
+> Installation guide link has been added to each dependency
+- [Git](https://git-scm.com/) - using a GUI such as [SourceTree](https://www.sourcetreeapp.com/) or [GitHub Desktop](https://desktop.github.com/) can help
+- [RVM](https://rvm.io/rvm/install) 
+- Ruby 3.2.1
+- [Redis 7.0 [atleast]](https://redis.io/docs/getting-started/installation/install-redis-on-linux/)
+- [ImageMagick](https://imagemagick.org/) - Image manipulation library
+- [Node.js 16.x](https://nodejs.org/it/download)
+- [Yarn](https://yarnpkg.com/getting-started/install)
+- [CMAKE](https://cmake.org/install/)
+- OpenSSL
+- libpq-dev
+- [PostgreSQL](https://www.postgresql.org/) (`12`) - Database
+
```

Comment:
Add `apt` command to download all dependencies for ubuntu and debian, For ruby use `rvm` or `asdf` guide to install the correct version of ruby

Reply:
updated linux documentation with respective commands of each dependency